### PR TITLE
Cli avmtestnet

### PR DESCRIPTION
**Pull request by beidouz**
## Notice

It is not allowed to submit your PR to the master branch directly, please submit your PR to the master-pre-merge branch.

## 

Please include a brief summary of the change that this pull request proposes. Include any relevant motivation and context. List any dependencies required for this change.

- Include avmtestnet as a new cli network option.
- Temporarilly using 31 as the avmtestnet chainid

Fixes Issue # .

## Type of change

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [ ] Bug fix.
- [ ] New feature.
- [ ] Enhancement.
- [ ] Unit test.
- [ ] Breaking change (a fix or feature that causes existing functionality to not work as expected).
- [ ] Requires documentation update.

## Testing

Please describe the tests you used to validate this pull request. Provide any relevant details for test configurations as well as any instructions to reproduce these results.

-

## Verification

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [x] I have self-reviewed my own code and conformed to the style guidelines of this project.
- [ ] New and existing tests pass locally with my changes.
- [x] I have added tests for my fix or feature.
- [ ] I have made appropriate changes to the corresponding documentation.
- [x] My code generates no new warnings.
- [x] Any dependent changes have been made.

### CODE REVIEW

File: config/avmtestnet/genesis.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    }
+  },
+  "networkBalanceAlloc": {
+    "0": {
+      "balance": "465934586660000000000000000"
+    }
+  },
+  "energyLimit": "15000000",
+  "nonce": "0x00",
+  "difficulty": "0x4000",
```
Comment (AionJayT) [inline config/avmtestnet/genesis.json]: difficulty set too high

**Comment by aionick** [comment]
Just a note, these changes have been `cherry-pick`'ed into #761. If any additional changes need to be made we can bring them over, but otherwise if all is good and ready to be merged it would be better to just delete this branch. We had to bring them over because we needed the changes to get the test net up and running.

File: config/avmtestnet/genesis.json (hunk 1 above)
Comment (beidouz) [reply config/avmtestnet/genesis.json]: difficulty is lowered to 0x0008 in #761

**Comment by AlexandraRoatis** [comment]
@AionJayT the `config.xml` and `genesis.json` are only placeholders. The files will be made public only when the test network is.

**Comment by AlexandraRoatis** [comment]
Closing this PR since the commits are already in #761.

**Pull request closed without merging**